reascripts/ReaSpeech/source/ASRControls.lua

```diff
@@ -0,0 +1,151 @@
+--[[
+
+ASRControls.lua - Controls/configuration for ASR plugin
+
+]]--
+
+ASRControls = PluginControls {
+  DEFAULT_TAB = 'asr-simple',
+
+  DEFAULT_LANGUAGE = '',
+  DEFAULT_MODEL_NAME = 'small',
+
+  SIMPLE_MODEL_SIZES = {
+    {'Small', 'small'},
+    {'Medium', 'medium'},
+    {'Large', 'distil-large-v3'},
+  },
+
+  tabs = function(self)
+    return {
+      ReaSpeechPlugins.tab('asr-simple', 'ASR - Simple',
+        function() self:render_simple() end),
+      ReaSpeechPlugins.tab('asr-advanced', 'ASR - Advanced',
+        function() self:render_advanced() end),
```

Comment:
> "ASR" is a bit of jargon that I'd prefer not to burden users with. In addition, I feel like these belong as sub-tabs (or some other UI), if we are going to use tabs to represent different types of tasks.
> 
> I could see moving the "Process Items" and "Label Track Languages" so that they're part of these tabbed forms, that way the available actions can depend on which main tab is selected.

i hear ya re: "ASR". i'd considered changing the "Process *" buttons to instead read "Transcribe *" and call the tabs "Transcription". good idea re: contextual buttons, i'm definitely open to that.